Services/data/devicemodels/cab-01-protobuf.json

```diff
@@ -0,0 +1,51 @@
+{
+  "SchemaVersion": "1.0.0",
+  "Id": "cab-01-proto",
+  "Version": "0.0.1",
+  "Name": "Cab 01 Proto",
+  "Description": "Cab data on Protobuff",
```

Comment:
typo: "protobuff" -> "protobuf". Also would rephrase to "Cab prototype telemetry serialized with protocol buffers"